consumers/writers/influxdb/consumer.go

```diff
@@ -21,65 +23,62 @@ var errSaveMessage = errors.New("failed to save message to influxdb database")
 
 var _ consumers.Consumer = (*influxRepo)(nil)
 
+type RepoConfig struct {
+	Bucket string
+	Org    string
+}
+
 type influxRepo struct {
-	client influxdata.Client
-	cfg    influxdata.BatchPointsConfig
+	client influxdb2.Client
+	cfg    RepoConfig
 }
 
 // New returns new InfluxDB writer.
-func New(client influxdata.Client, database string) consumers.Consumer {
+func New(client influxdb2.Client, config RepoConfig) consumers.Consumer {
 	return &influxRepo{
 		client: client,
-		cfg: influxdata.BatchPointsConfig{
-			Database: database,
-		},
+		cfg:    config,
 	}
 }
 
 func (repo *influxRepo) Consume(message interface{}) error {
-	pts, err := influxdata.NewBatchPoints(repo.cfg)
-	if err != nil {
-		return errors.Wrap(errSaveMessage, err)
-	}
+	var err error
+	var pts []*write.Point
 	switch m := message.(type) {
 	case json.Messages:
-		pts, err = repo.jsonPoints(pts, m)
+		pts, err = repo.jsonPoints(m)
 	default:
-		pts, err = repo.senmlPoints(pts, m)
+		pts, err = repo.senmlPoints(m)
 	}
 	if err != nil {
 		return err
 	}
-
-	if err := repo.client.Write(pts); err != nil {
-		return errors.Wrap(errSaveMessage, err)
-	}
-	return nil
+	writeAPI := repo.client.WriteAPIBlocking(repo.cfg.Org, repo.cfg.Bucket)
```

Comment:
I am having doubt on this variable naming... I would probably use `wa` instead of `writeAPI`, Go idiomatic is using simple names, especially if the code is near and readable... But it might be nitpicking.

@dborovcanin what do you think? There is a naming like this in several places.